.gitignore

```diff
@@ -21,3 +21,5 @@ visor
 build/.*
 support/tools/bin
 
+# Deployment
+*.env
```

Comment:
`*.env` will not match `.env` files in subfolders. `.env` would match all `.env` files in all paths in the git project.

See: https://www.atlassian.com/git/tutorials/saving-changes/gitignore

Reply:
I tested it in local, moreover, the the doc you provided, I saw it will match the file in subfolders.
<img width="568" alt="截圖 2023-05-11 下午3 11 10" src="https://github.com/filecoin-project/lily/assets/4417105/fb5df84e-4160-4759-b405-6df08f745848">

Reply:
guess we do want to filter out files like `abc.env` in the future.